### PR TITLE
Jenkins mac buildbot

### PR DESCRIPTION
**Pull request by slefrancois**
Should pass without errors once #6147 is merged.

### CODE REVIEW

**Comment by lamblin** [comment]
Jenkins test mac

**Comment by slefrancois** [comment]
@lamblin try again, didn't trigger

**Comment by slefrancois** [comment]
jenkins test mac

File: .jenkins/jenkins_buildbot_mac.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+export PATH="/Users/jenkins/miniconda2/bin:/usr/local/bin:$PATH"
+
+# CUDA
+export PATH=/usr/local/cuda/bin${PATH:+:${PATH}}
+export DYLD_LIBRARY_PATH=/usr/local/cuda/lib\
+                         ${DYLD_LIBRARY_PATH:+:${DYLD_LIBRARY_PATH}}
+
+# Build libgpuarray
+GPUARRAY_CONFIG="Release"
+DEVICE=cuda
```
Comment (abergeron) [inline .jenkins/jenkins_buildbot_mac.sh]: If you're not running libgpuarray/pygpu tests, this is useless.

**Comment by abergeron** [comment]
You might still want to wait for #6170

**Comment by slefrancois** [comment]
jenkins test mac

**Comment by slefrancois** [comment]
jenkins test mac

**Comment by slefrancois** [comment]
jenkins test mac

**Comment by slefrancois** [comment]
jenkins test mac

**Comment by slefrancois** [comment]
jenkins test mac

**Pull request merged**